requirements_test.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+
+pep8>=1.7.0
+nose>=1.3.7
```

Comment:
The line `-r requirements.txt` pulls in the base requirements (librosa and any others added in the future) from the requirements.txt file. I should probably add a clarifying comment.